### DISCUSSION TITLE
Migrate CRA to Farm

### DISCUSSION THREAD

**Discussion by khangviet1996** [farm-fe/farm#1639] [Q&A]
After I migrated to Vite, I've encountered issues like a large number of requests. So, I want to try migrating to Farm. Do you have a link or guide for reference? I would appreciate it ❤️

My stack: React, Antd@4.20, Less, ...

**Comment by wre232114** [comment]
1. transform your vite config to farm config. for example vite base to farm output.publicPath
2. move all vite plugins to vitePlugins of farm.config.ts
3. add farm react/less/sass/postcss  plugin if your project needed to plugins of farm.config.ts
